frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -103,13 +206,38 @@ const Input = (props: TaipyInputProps) => {
 
     return (
         <Tooltip title={hover || ""}>
-            <TextField
+            <StyledTextField
                 margin="dense"
                 hiddenLabel
                 value={value ?? ""}
                 className={className}
                 type={type}
                 id={id}
+                inputProps={
+                    type !== "text"
+                        ? {
+                              step: step ? step : 1,
+                              min: min,
+                              max: max,
+                          }
+                        : {}
+                }
+                InputProps={
+                    type !== "text"
+                        ? {
+                              endAdornment: (
+                                  <>
```

Comment:
I suppose these are displayed side by side ?
If so, can you display them one over the other ?

Reply:
resolved